guides/advanced_guides/snapshots_and_dumps.md

```diff
@@ -41,17 +51,17 @@ During snapshot creation, snapshots are **overwritten**. Meaning, only the last
 
 Because snapshots are zipped copies of your database that did not go through any processing, starting a MeiliSearch instance from one of them is significantly faster.
```

Comment:
```suggestion
Because snapshots are hard copies of your database that did not go through any processing, starting a MeiliSearch instance from one of them is significantly faster than dumps.
```

Or should we say `as fast as starting MeiliSearch without a snapshot`?